docs/search-views/fulltext-search/queries/index.md

```diff
@@ -3,3 +3,126 @@ sidebar_position: 40
 title: Fulltext Search Queries
 ---
 
+With Macrometa, the `SEARCH` keyword is used to filter your queries using a search view. By employing this keyword, you can:
+
+- Filter documents based on Boolean expressions and functions in [C8QL](../../../queries/c8ql/).
+- Match documents that reside in different collections.
+- Sort the resulting set based on how closely each document matches the search conditions.
+
+## Syntax
+
+The `SEARCH` statement must be used within a `FOR...IN` operation. For the correct sequence, the `SEARCH` statement should be placed after `FOR` and before any additional operations.
+
+```sql
```

Comment:
should this be `c8ql`? I think it should be but not sure whether SQL supports `SEARCH`!
cc @martin-macrometa